### PR TITLE
Update SOFTWARE-LIST.md with Snow Software information

### PR DESCRIPTION
**Pull request by AlastairPooley**
Snow Software added

## ✅ Pre-approval checklist ##

- [ ] This PR has an informative and human-readable title.
- [ ] Changes are limited to a single goal - _eschew scope creep!_
- [ ] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [ ] All relevant type-of-change labels have been added.
- [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

### CODE REVIEW

**Review by iainDe — DISMISSED** [review]

File: SOFTWARE-LIST.md [hunk 1]
```diff
@@ -2403,6 +2403,8 @@ download | | 12/20/2021 |
 | SmileCDR | | | | | [SmileCDR Blog Post](https://www.smilecdr.com/our-blog/a-statement-on-log4shell-cve-2021-44228) | | | |
 | Snakemake | Snakemake | 6.12.1 | Not Affected |  | [https://snakemake.readthedocs.io/en/stable/](https://snakemake.readthedocs.io/en/stable/) | | | 12/21/2021 |
 | Sn0m | | | | | [Sn0m Link](https://www.snom.com/en/press/log4j-poses-no-threat-snom-phones/) | | | |
+| Snow Software | Snow Commander | 8.1 to 8.10.2 | Fixed | Yes |[Snow Software Commmunity Link](https://community.snowsoftware.com/s/feed/0D5690000B4U6hUCQS) | | | |
+| Snow Software | VM Access Proxy | v3.1 to v3.6 " Fixed | Yes | [Snow Software Commmunity Link](https://community.snowsoftware.com/s/feed/0D5690000B4U6hUCQS) | | | |
```
Comment (iainDe) [inline SOFTWARE-LIST.md]: Lines should read as the following:
| Snow Software | Snow Commander | 8.1 to 8.10.2 | Affected | Yes | [Snow Software Community Link](https://community.snowsoftware.com/s/feed/0D5690000B4U6hUCQS) | | | | 
| Snow Software | VM Access Proxy | v3.1 to v3.6 I Affected | Yes | [Snow Software Community Link](https://community.snowsoftware.com/s/feed/0D5690000B4U6hUCQS) | | | |

**Review by iainDe — CHANGES_REQUESTED** [review]
``

**Review by AlastairPooley — COMMENTED** [review]
Thanks iainDe

**Review by justmurphy — APPROVED** [review]
Thank you for the submission.

**Review by iainDe — APPROVED** [review]

**Pull request merged**